src/gui/pedalboard/cables/Cable.cpp

```diff
@@ -54,17 +92,58 @@ auto Cable::createCablePath (juce::Point<float> start, juce::Point<float> end)
     return std::move (bezierPath);
 }
 
+void Cable::checkNeedsRepaint (bool force)
```

Comment:
I'm not sure `checkNeedsRepaint()` is quite the right name for this method... maybe something more like `repaintIfNeeded()`?